pkg/util/startuplogger/startuplogger.go

```diff
@@ -0,0 +1,81 @@
+package startuplogger
+
+import (
+	"encoding/json"
+	"os"
+	"time"
+
+	"k8s.io/klog/v2"
+)
+
+type ServiceData struct {
+	Name         string    `json:"name"`
+	Dependencies []string  `json:"dependencies"`
+	Start        time.Time `json:"start"`
+	Ready        time.Time `json:"ready"`
+}
+
+func (s ServiceData) MarshalJSON() ([]byte, error) {
+	type Alias ServiceData
+	return json.Marshal(struct {
+		Alias
+		StartupTime string `json:"duration"`
+	}{
+		Alias:       (Alias)(s),
+		StartupTime: s.Ready.Sub(s.Start).String(),
+	})
+}
```

Comment:
Is this necessary? Can we add StartupTime to the ServiceData?
Also, would `timeToReady` would be more descriptive? Both for the field name and json tag